gapis/atom/group_test.go

```diff
@@ -45,9 +45,9 @@ var tree = `Group 'root' [0..1099]
  │                └─ [0..99] ───── Atoms [500..599]
  └─ [303..702] ── Atoms [600..999]`
 
-func buildTestGroup() Group {
+func buildTestGroup(end uint64) Group {
```

Comment:
Done (I was actually thinking the same, once I understood why there was a difference...)